src/Sentry/SentrySession.cs

```diff
@@ -36,6 +36,13 @@ public class SentrySession : ISentrySession
     // Start at -1 so that the first increment puts it at 0
     private int _sequenceNumber = -1;
 
+    private int _isMarkedAsPendingUnhandled;
```

Comment:
```suggestion
    private InterlockedBoolean _isMarkedAsPendingUnhandled;
```

See: https://github.com/getsentry/sentry-dotnet/pull/4585